Formula/c7n.rb

```diff
@@ -0,0 +1,121 @@
+    sha256 "28669905fe725965daa16184933676547c5bb40a5153055a8dee2a4bd7933ad3"
+  end
+
+  resource "python-dateutil" do
+    url "https://files.pythonhosted.org/packages/be/ed/5bbc91f03fa4c839c4c7360375da77f9659af5f7086b7a7bdda65771c8e0/python-dateutil-2.8.1.tar.gz"
+    sha256 "73ebfe9dbf22e832286dafa60473e4cd239f8592f699aa5adaf10050e6e1823c"
+  end
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/64/c2/b80047c7ac2478f9501676c988a5411ed5572f35d1beff9cae07d321512c/PyYAML-5.3.1.tar.gz"
+    sha256 "b8eac752c5e14d3eca0e6dd9199cd627518cb5ec06add0de9d32baeee6fe645d"
+  end
+
+  resource "s3transfer" do
+    url "https://files.pythonhosted.org/packages/50/de/2b688c062107942486c81a739383b1432a72717d9a85a6a1a692f003c70c/s3transfer-0.3.3.tar.gz"
+    sha256 "921a37e2aefc64145e7b73d50c71bb4f26f46e4c9f414dc648c6245ff92cf7db"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/21/9f/b251f7f8a76dec1d6651be194dfba8fb8d7781d10ab3987190de8391d08e/six-1.14.0.tar.gz"
+    sha256 "236bdbdce46e6e6a3d61a337c0f8b763ca1e8717c03b369e87a7ec7ce1319c0a"
+  end
+
+  resource "tabulate" do
+    url "https://files.pythonhosted.org/packages/57/6f/213d075ad03c84991d44e63b6516dd7d185091df5e1d02a660874f8f7e1e/tabulate-0.8.7.tar.gz"
+    sha256 "db2723a20d04bcda8522165c73eea7c300eda74e0ce852d9022e0159d7895007"
+  end
+
+  resource "urllib3" do
+    url "https://files.pythonhosted.org/packages/05/8c/40cd6949373e23081b3ea20d5594ae523e681b6f472e600fbc95ed046a36/urllib3-1.25.9.tar.gz"
+    sha256 "3018294ebefce6572a474f0604c2021e33b3fd8006ecd11d62107a5d2a963527"
+  end
+
+  resource "zipp" do
+    url "https://files.pythonhosted.org/packages/ce/8c/2c5f7dc1b418f659d36c04dec9446612fc7b45c8095cc7369dd772513055/zipp-3.1.0.tar.gz"
+    sha256 "c599e4d75c98f6798c509911d08a22e6c021d074469042177c8c86fb92eefd96"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    # trim last decimal point version to match semver returned from version command
+    cli_version = version.to_s.split(".")[0..2].join(".")
+    assert_match cli_version, shell_output("#{bin}/custodian version")
+
+    (testpath/"policy.yml").write <<~EOF
+      policies:
+      - name: ec2-auto-tag-user
+        resource: ec2
+        mode:
+          type: cloudtrail
+          role: arn:aws:iam::{account_id}:role/custodian-auto-tagger
+          # note {account_id} is optional. If you put that there instead of
+          # your actual account number, when the policy is provisioned it
+          # will automatically inherit the account_id properly
+          events:
+            - RunInstances
+        filters:
+          - tag:CreatorName: absent
+        actions:
+          - type: auto-tag-user
+            tag: CreatorName
+            principal_id_tag: CreatorId
+    EOF
+    output = shell_output("custodian validate #{testpath}/policy.yml")
```

Comment:
You should check the output of this.

Reply:
There isn't any checkable output for that except its error code which is already checked by `brew test` command.

Should I just omit the `output` variable ?

Reply:
@SMillerDev thoughts?

Reply:
@SMillerDev ping

Reply:
Can't you make the validation fail for some output? I think a correct error might be safer than just asserting on the status.

Reply:
@SMillerDev I pushed up a change to test both the valid case and an invalid case. How do I correctly do a negative test? The `assert_match` is failing because the command returns an expected non-zero error code due to the invalid configuration.

Reply:
https://rubydoc.brew.sh/Homebrew/Assertions.html#shell_output-instance_method

`shell_output` accepts a second (optional) parameter that is an exit status to match against.

Reply:
Thank you @gromgit! That fixed it.